app/javascript/onboarding/components/PersonalInfoForm.jsx

```diff
@@ -99,27 +101,34 @@ class PersonalInfoForm extends Component {
     const { prev } = this.props;
     return (
       <div className="onboarding-main">
-        <div className="onboarding-content about">
-          <h2>About You!</h2>
+        <Navigation prev={prev} next={this.onSubmit} />
+        <div className="onboarding-content">
+          <header className="onboarding-content-header">
+            <h1 className="title">
+              About You
+            </h1>
+          </header>
           <form>
             {setupFormTextField({
               labelText: 'Where are you located?',
               entityName: 'location',
+              placeholderText: 'e.g. New York, NY',
```

Comment:
Curious to know if we actually _need_ the `e.g.` here? IMO, it's a bit repetitive, since the placeholder text disappears when you start typing into the input.

Reply:
We probably don't need `e.g.` so I can remove this.